CHANGES.md

```diff
@@ -4,6 +4,7 @@
 ## Unreleased
 - Added/reactivated documentation as `sqlalchemy-cratedb`
 - Unlocking supporting timezone-aware `DateTime` fields
+- Accept marshalling Python `datetime.date` values on `sa.DateTime` fields
```

Comment:
Please use past tense, maybe something like:
`Added support for....`

Reply:
Improved wording. Thanks.